sw/ground_segment/cockpit/live.ml

```diff
@@ -406,6 +414,16 @@ let get_speech_name = fun af_xml def_name ->
     fvalue "SPEECH_NAME" default_speech_name
   with _ -> default_speech_name
 
+let get_cam_aov = fun af_xml ->
```

Comment:
Do you really need the cam field of view in the GCS since you are doing the computation in server agent ? It doesn't seemed to be used here anymore. Some question for roll, pitch, etc

Reply:
Thanks for the review! I will clean up all of my code and removed any extra variables/functions I no longer need.